jwst/combine_1d/combine1d.py

```diff
@@ -596,22 +619,32 @@ def check_exptime(exptime_key):
     return exptime_key
 
 
-def do_correction(asn_file, exptime_key, interpolation):
+def do_correction(asn_file, exptime_key, interpolation, background=False):
     """Combine the input spectra.
 
     Parameters
     ----------
-    asn_file: str
+    asn_file : str
         The association file name.
 
-    exptime_key: str
+    exptime_key : str
         A string identifying which keyword to use to get the exposure time,
         which is used as a weight when combining spectra.
 
-    interpolation: str
+    interpolation : str
         The type of interpolation between input pixels.
+
+    background : bool, default=False
+        If the flux data are actually background rather than a target
+        spectrum, `background` should be set to True.  In this case, the
+        values read from the flux column of each input spectrum will be
+        divided by the npixels column (if that column exists).  This is
+        to convert the values to background per pixel.
     """
 
+    log.debug("Using exptime_key = {}.".format(exptime_key))
+    log.debug("The FLUX data will be treated as background data.")
```

Comment:
Shouldn't this message be part of an if-block that depends on the value of `background`? Right now it looks like it's always printed (whenever debug is enabled).